src/module_3/push_notifications.py

```diff
@@ -0,0 +1,36 @@
+import pandas as pd
+from typing import Any
+import logging
+
+CSV_PATH = "/home/manucorujo/zrive-data/feature_frame.csv"
```

Comment:
Use here relative paths. Global paths are always a bad practice, since this code cannot be directly exported.